compiler/luci/pass/include/luci/Pass/ForceQuantParamPass.h

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_FORCE_QUANT_PARAM_PASS_H__
+#define __LUCI_FORCE_QUANT_PARAM_PASS_H__
+
+#include <loco.h>
+
+#include <logo/Pass.h>
+
+using TensorVector = std::vector<std::string>;
+using ScaleVector = std::vector<float>;
+using ZPVector = std::vector<int64_t>;
```

Comment:
These names, especially `TensorVector` is generic that may slip in files that include this.
How about using in another namespace,... maybe `luci::pass` ?

Reply:
I've moved them inside `ForceQuantParamPass`, because they are used only in this pass.